Projects/Market/Sources/Market.swift

```diff
@@ -21,11 +21,14 @@ public enum Market: String, CaseIterable, Codable {
     }
 
     public var title: String {
-        switch self {
-        case .norway: return "Norge"
-        case .sweden: return "Sverige"
-        case .denmark: return "Danmark"
-        case .france: return "France"
+        switch (self, Localization.Locale.currentLocale) {
+        case (.sweden, .sv_SE): return "Sverige"
```

Comment:
Why not load from Lokalise instead?

Reply:
Didn't realise they were present in Lokalise already. It's done now!